### ISSUE TITLE
Hotkeys dont work, while im in a game 

### ISSUE THREAD

**Issue by PattiXY** [G-Stas/Volumey#46]
Hi,
when I play a game the hotkeys for Spotify wont work. I already tried starting it in Admin or giving it more Priorities, but nothing worked. I also dont know if there is any chance to do this, because I think that the games dont allow these hotkeys because of AntiCheat or something else. I hope you can help me with this problem. But the programm overall is realy nice. Or is there a Problem with the Microsoft store version?

**Comment by G-Stas** [comment]
Yeah, I have encountered the same problem in several games. I think It's just some games absorb all the input completely and prevent it going to the OS somehow. 
Using keyboard hook to capture hotkeys instead of the Windows API might help with the problem, but last time I was trying to implement this it wasn't quite stable, but I'll look into it.
